Source/Softswitch/src/softswitch.cpp

```diff
@@ -47,7 +49,7 @@ void softswitch_trivial_log_handler(const char* &logStr)
                                 // has not actually been sent.
     hdr->swAddr = P_SW_MOTHERSHIP_MASK | P_SW_CNC_MASK;
     hdr->swAddr |= ((P_CNC_LOG << P_SW_OPCODE_SHIFT) & P_SW_OPCODE_MASK);
-    hdr->pinAddr = tinselId(); // usurp Pin Addr for the source HW addr
+    hdr->pinAddr = src;         // The index of the device on the supervisor
```

Comment:
Good stuff

Reply:
I have not implemented the needed changes in Mothership yet!

Reply:
That's fine, because they're not currently used (?, he asks tentatively?) :sweet_potato:

Reply:
They are used for the log handler messages, but the current implementation will just report incorrect thread/core.

Reply:
Will either modify logserver message or implement needed changes in mothership to avoid user confusion.